### PR TITLE
[DPC-4334] Government System Banner / System Use Agreement

### PR DESCRIPTION
**Pull request by chris-ronning-ny**
## 🎫 Ticket

https://jira.cms.gov/browse/DPC-4334

## 🛠 Changes

- Create a static page for the System Use Agreement. I separated this from the main `application.html.erb` in order to simplify some page style changes -- namely, spacing and background color. This allows me to match the style specified in the image on the ticket.
- Add a toggle to remove the bottom border from the header, in order to match the style specified in the image on the ticket.
- Create the link and text to the system use agreement and add it to the invitation login and the main login page.

## ℹ️ Context

We want to link users to a government system use agreement when they are prompted to login.

## 🧪 Validation

- [ ] Manual testing
- [ ] Unit testing

System Use Agreement
Screenshot 2024-10-21 at 1 52 29 PM
Screenshot 2024-10-21 at 1 52 21 PM

### CODE REVIEW

File: dpc-portal/app/components/core/navigation/system_use_agreement_link_component.html.erb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<div class="font-body-2xs margin-bottom-3">
+  <span>
+    By signing in with Login.gov, you are agreeing to our
+    <%= link_to system_use_agreement_path do %>
```
Comment (jdettmannnava) [inline dpc-portal/app/components/core/navigation/system_use_agreement_link_component.html.erb:4]: Needs a target to open in a new tab/window

**Review by jdettmannnava — COMMENTED** [review]
Everything else looks great!

**Comment by chris-ronning-ny** [comment]
@jdettmannnava added target: :_blank, and it opens in a new tab now.

**Review by jdettmannnava — APPROVED** [review]
LGTM

**Pull request merged**